src/tutorials/sse/echo/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
```

Comment:
`echo` has a meaning of reflecting back what was sent, but this is an `sse` `proxy` example, not `sse` `echo`.
Suggest we rename directory structure to `sse/proxy`.